promscale/cli.md

```diff
@@ -48,35 +47,49 @@ You can also find information on flags with `promscale_<version> -help`.
 | metrics.cache.exemplar.size | unsigned-integer | 10000 | Maximum number of exemplar metrics key-position to cache. It has one-to-one mapping with number of metrics that have exemplar, as key positions are saved per metric basis. |
-| metrics.cache.labels.size | unsigned-integer | 10000 | Maximum number of labels to cache. |
+| metrics.cache.labels.size | unsigned-integer | 10000 | Maximum number of labels to cache.|
 | metrics.cache.metrics.size | unsigned-integer | 10000 | Maximum number of metric names to cache. |
-| metrics.cache.series.initial-size | unsigned-integer| 250000 | Initial number of elements in the series cache. |
-| metrics.cache.series.max-bytes | unsigned-integer or percentage | 50% |  Target for amount of memory to use for the series cache. Specified in bytes or as a percentage of the memory-target (for example, 50%). |
-| metrics.high-availability | boolean | false | Enable external_labels based HA. |
+| metrics.cache.series.initial-size | unsigned-integer | 250000 | Initial number of elements in the series cache. |
+| metrics.cache.series.max-bytes | unsigned-integer or percentage | 50% | Target for amount of memory to use for the series cache. Specified in bytes or as a percentage of the memory-target. For example, 50%.|
+| metrics.high-availability | boolean | false | Enable external_labels based HA.|
 | metrics.ignore-samples-written-to-compressed-chunks | boolean | false | Ignore/drop samples that are being written to compressed chunks. Setting this to false allows Promscale to ingest older data by decompressing chunks that were earlier compressed. However, setting this to true will save your resources that may be required during decompression. |
 | metrics.multi-tenancy | boolean | false | Use multi-tenancy mode in Promscale. |
 | metrics.multi-tenancy.allow-non-tenants | boolean | false | Allow Promscale to ingest/query all tenants as well as non-tenants. By setting this to true, Promscale will ingest data from non multi-tenant Prometheus instances as well. If this is false, only multi-tenants (tenants listed in 'multi-tenancy-valid-tenants') are allowed for ingesting and querying data. |
-| metrics.multi-tenancy.valid-tenants | string | allow-all |  Sets valid tenants that are allowed to be ingested/queried from Promscale. This can be set as: 'allow-all' (default) or a comma separated tenant names. 'allow-all' makes Promscale ingest or query any tenant from itself. A comma separated list will indicate only those tenants that are authorized for operations from Promscale. |
+| metrics.multi-tenancy.valid-tenants | string | allow-all | Sets valid tenants that are allowed to be ingested/queried from Promscale. This can be set as: 'allow-all' (default) or a comma separated tenant names. 'allow-all' makes Promscale ingest or query any tenant from itself. A comma separated list will indicate only those tenants that are authorized for operations from Promscale. |
```

Comment:
```suggestion
| metrics.multi-tenancy.valid-tenants | string | allow-all | Sets valid tenants that are allowed to be ingested/queried from Promscale. This can be set as 'allow-all' (default), or as comma separated tenant names. 'allow-all' makes Promscale ingest or query any tenant from itself. A comma separated list indicates only those tenants that are authorized for operations from Promscale. |
```